<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>727</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Malgun Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="depCity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>수서</string>
    </property>
    <item>
     <property name="text">
      <string>수서</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>동탄</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>지제</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>천안아산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>오송</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>익산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>정읍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>광주송정</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>나주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>목포</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>대전</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>김천구미</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>동대구</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>신경주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>울산(통도사)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>부산</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="arrCity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>수서</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>동탄</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>지제</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>천안아산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>오송</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>익산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>정읍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>광주송정</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>나주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>목포</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>대전</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>김천구미</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>동대구</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>신경주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>울산(통도사)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>부산</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="depHour">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="searchSeat">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="depDate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>7</month>
      <day>19</day>
     </datetime>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="displayFormat">
     <string>yyyy.MM.dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="resTable">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>96</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>23</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>기차</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>출발/시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>도착/시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>좌석</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="tryReservation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>예약하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>map.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="checkBoxes">
     <item>
      <widget class="QCheckBox" name="checkBox_01">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_02">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_03">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_04">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_05">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_06">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_07">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_08">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_09">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>예약</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>5</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>출발</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>5</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>도착</string>
    </property>
   </widget>
   <widget class="QComboBox" name="route_map">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>SRT 노선도</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (경부선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (호남선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (경전선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (전라선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (강릉선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (중앙선)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KTX 노선도 (중부내륙선)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SRT/KTX 노선도</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>78</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="check_str">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SRT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_ktx">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>KTX</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
